src/fu-security-attr.c

```diff
@@ -207,6 +207,314 @@ fu_security_attr_get_name(FwupdSecurityAttr *attr)
+			 "device memory from being accessed by less secure programs.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_ENCRYPTED_RAM) == 0) {
+		/* TRANSLATORS: longer description */
+		return _(
+		    "Encrypted RAM makes it impossible for information that is stored in device "
+		    "memory to be read, if the memory chip is removed and accessed.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_IOMMU) == 0) {
+		/* TRANSLATORS: longer description */
+		return _("IOMMU protection prevents connected devices from accessing unauthorized "
+			 "parts of system memory.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_KERNEL_LOCKDOWN) == 0) {
+		/* TRANSLATORS: longer description */
+		return _("Linux Kernel Lockdown mode prevents administrator (root) accounts from "
+			 "accessing and changing critical parts of system software.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_KERNEL_TAINTED) == 0) {
+		/* TRANSLATORS: longer description */
+		return _(
+		    "Linux Kernel Verification makes sure that critical system software has "
+		    "not been  tampered with. Using device drivers which are not provided with the "
+		    "system can  revent this security feature from working correctly.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_KERNEL_SWAP) == 0) {
+		/* TRANSLATORS: longer description */
+		return _("Kernel Swap temporarily saves information to disk as you work. This "
+			 "information which could be accessed by someone if they gained access to "
+			 "the disk.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_SUSPEND_TO_RAM) == 0 ||
+	    g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_SUSPEND_TO_IDLE) == 0) {
+		/* TRANSLATORS: longer description */
+		return _(
+		    "During system sleep the device’s memory chip could be removed, allowing the "
+		    "information that is saved on it to be accessed.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_UEFI_PK) == 0 ||
+	    g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_UEFI_SECUREBOOT) == 0) {
+		/* TRANSLATORS: longer description */
+		return _("UEFI Secure Boot prevents malicious software from being loaded when the "
+			 "device starts.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_TPM_EMPTY_PCR) == 0 ||
+	    g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_TPM_RECONSTRUCTION_PCR0) == 0 ||
+	    g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_TPM_VERSION_20) == 0) {
+		return _(
+		    /* TRANSLATORS: longer description */
+		    "TPM (Trusted Platform Module) v2 is a computer chip which detects when the "
+		    "hardware components have been tampered with.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_MEI_MANUFACTURING_MODE) == 0 ||
+	    g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_PLATFORM_FUSED) == 0) {
+		/* TRANSLATORS: longer description */
+		return _("Manufacturing Mode is used during device assembly and disables security "
```

Comment:
Technically it doesn't disable them, it's _before_ they have been enabled.

Reply:
`Manufacturing Mode is used during device assembly when security features have yet to be enabled.` perhaps?

Reply:
Yup

Reply:
Fixed.